src/Security/Authentication/WsFederation/src/WsFederationOptions.cs

```diff
@@ -208,5 +209,5 @@ public TokenValidationParameters TokenValidationParameters
     /// <para>The default token handler for JsonWebTokens is a <see cref="JsonWebTokenHandler"/> which is faster than a <see cref="JwtSecurityTokenHandler"/>.</para>
     /// <para>There is an ability to make use of a Last-Known-Good model for metadata that protects applications when metadata is published with errors.</para>
     /// </remarks>
-    public bool UseTokenHandlers { get; set; }
+    public bool UseSecurityTokenHandlers { get; set; }
```

Comment:
Note: naming is slightly different than what was called out in API review because API review mistakenly used the name 'SecurityTokenValidators ' where for wsfed it is called 'SecurityTokenHandlers'

https://github.com/dotnet/aspnetcore/issues/49469

Reply:
which are not the `TokenHandler` unfortunately.
~~I wonder if, for the sake of simplicity and uniformity, we shouldn't keep UseSecurityTokenValidators even for WsFed ... just a suggestion.~~
Although looking at the code .. probably not.